etrago/cluster/snapshot.py

```diff
@@ -103,7 +132,42 @@ def tsam_cluster(timeseries_df, typical_periods=10, how='daily'):
     # get the origial Datetimeindex
     dates = timeseries_df.iloc[nrhours].index
 
-    return timeseries, cluster_weights, dates, hours
+    #get list of representative days
+    representative_day=[]
+
+    #cluster:medoid des jeweiligen Clusters
+    dic_clusterCenterIndices = dict(enumerate(clusterCenterIndices))
+    for i in clusterOrder:
+        representative_day.append(dic_clusterCenterIndices[i])
+
+    #get list of last hour of representative days
+    last_hour_datetime=[]
+    for i in representative_day:
+        last_hour = i * hours + hours - 1
+        last_hour_datetime.append(timeseries_df.index[last_hour])
+
+    #create a dataframe (index=nr. of day in a year/candidate)
+    df_cluster =  pd.DataFrame({
+                        'Cluster': clusterOrder, #Cluster of the day
+                        'RepresentativeDay': representative_day, #representative day of the cluster
+                        'last_hour_RepresentativeDay': last_hour_datetime}) #last hour of the cluster  
+    df_cluster.index = df_cluster.index + 1
+    df_cluster.index.name = 'Candidate'
+
+    #create a dataframe each timeseries (h) and its candiddate day (i) df_i_h
+    nr_day = []
+    x = len(timeseries_df.index)/hours+1
+
+    for i in range(1,int(x)):
+        j=1
+        while j <= hours: 
+            nr_day.append(i)
+            j=j+1
+    df_i_h = pd.DataFrame({'Timeseries': timeseries_df.index,
+                        'Candidate_day': nr_day})
+    df_i_h.set_index('Timeseries',inplace=True)
+
+    return df_cluster, cluster_weights, dates, hours, df_i_h
```

Comment:
the doubling of the return statement looks very weird to me. Intentional?

Reply:
I don't see another return statement (the one in 106 is deleted and a new one added in 170), can you help me?